src/validators/v1/default-rules.js

```diff
@@ -0,0 +1,177 @@
+const moment = require('moment')
+
+module.exports = {
+	create: (req) => {
+		try {
+			req.checkBody('type').isString().notEmpty().withMessage('Type is required and must be a string')
+
+			req.checkBody('target_field')
+				.isString()
+				.notEmpty()
+				.withMessage('Target field is required and must be a string')
+
+			req.checkBody('is_target_from_sessions_mentor')
+				.isBoolean()
+				.withMessage('is_target_from_sessions_mentor must be a boolean')
+
+			req.checkBody('requester_field')
+				.isString()
+				.notEmpty()
+				.withMessage('Requester field is required and must be a string')
+
+			req.checkBody('field_configs').optional().isJSON().withMessage('Field configs must be a valid JSON')
+
+			req.checkBody('matching_operator').isString().notEmpty().withMessage('Matching operator must be a string')
```

Comment:
@nevil-mathew Can you validate allowed operators only ?

Reply:
@sumanvpacewisdom Please look into this